src/data/dapp_promotions.json

```diff
@@ -47,4 +47,10 @@
     "link": "https://app.moonfit.xyz/astar-reward",
     "img": "images/dapp_promotions/moonfit.png"
   }
+{
+    "name": "Archisinal",
+    "shortDescription": "Pioneering Real-World-Assets in the built environment",
+    "link": "https://www.archisinal.io/",
+    "img": "images/dapp_promotions/Archisinal Logo.png"
```

Comment:
Hi Tom, Thanks for the contributing. I just want to make same filename with others.

so, can you update image name as `archisinal.png` ?